src/flavors/columbia/config.json

```diff
@@ -1706,7 +1706,7 @@
           },
           {
             "start_field_index": 62,
-            "end_field_index": 73,
+            "end_field_index": 71,
```

Comment:
I was getting an "index out of bounds" error when parsing the `fields` array - it seems like we ignore this field, or silently fail, when this number is greater than the length of the array?

I double checked, and I think these new numbers are correct.

Reply:
I think that's right-- the very last index is not enforced. As long as the final item visible is the submit button with the new indices it should be fine.